### PR TITLE
Employee's anniverasy module

### PR DESCRIPTION
**Pull request by MarkingSC**
This module allows to use employment anniversary inside work information tab.

Note: The field is not computed by contract start date. It is just for informative purposes.

### CODE REVIEW

File: hr_employee_anniversary/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrEmployee(models.Model):
+
+    _inherit = "hr.employee"
+
+    # Anniversaries fields
```
Comment (eilst) [inline hr_employee_anniversary/models/hr_employee.py]: unnecessary comment, the field name is self-explanatory

File: hr_employee_anniversary/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+from . import models
```
Comment (eilst) [inline hr_employee_anniversary/__init__.py]:
```suggestion
from . import models

```

File: hr_employee_anniversary/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+from . import hr_employee
```
Comment (eilst) [inline hr_employee_anniversary/models/__init__.py]:
```suggestion
from . import hr_employee

```

**Review by eilst — APPROVED** [review]

File: hr_employee_anniversary/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        Adds a employment anniversary field next to birthday field.
+    """,
+
+    'author': "BerrySoft MX, Odoo Community Association (OCA)",
+    'website': "https://github.com/OCA/hr",
+
+    # Categories can be used to filter modules in modules listing
+    # for the full list
+    'category': 'Generic Modules/Human Resources',
+    'version': '0.1',
```
Comment (nikul-serpentcs) [inline hr_employee_anniversary/__manifest__.py]: @MarkingSC IMO `'version': '0.1'`, -->` 'version': '12.0.1.0.0',` or If new module then ` 'version': '12.0.0.0.0',`

File: hr_employee_anniversary/views/hr_employee_views.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+
+<odoo>
+
+  <data>
```
Comment (nikul-serpentcs) [inline hr_employee_anniversary/views/hr_employee_views.xml]: no need to `data` tag

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

File: hr_employee_anniversary/__manifest__.py (hunk 4 above)
Comment (MarkingSC) [reply hr_employee_anniversary/__manifest__.py]: Thanks, I will solve that.

**Comment by gurneyalex** [comment]
CLA is OK, I think. Problem could be comming from the commits on behalf of @eilst but he seems to have an ECLA so I'm not sure...

File: hr_employee_anniversary/views/hr_employee_views.xml (hunk 5 above)
Comment (MarkingSC) [reply hr_employee_anniversary/views/hr_employee_views.xml]: Done, thanks.

**Comment by jarroyomorales** [comment]
I think `hr_employee_seniority` already adds this feature. It allows you to set an initial employment date and also computes how long you have been working with the company. The module is on 11.0 but I already did the migration PR: https://github.com/OCA/hr/pull/606 to 12.0. If we are not talking about the same feature please tell me.

**Comment by MarkingSC** [comment]
@jarroyomorales you are right. It's already satisfied with that module. Thanks.

**Pull request closed without merging**